logstash.go

```diff
@@ -74,6 +74,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 			Hostname: m.Container.Config.Hostname,
 		}
 
+		tags := GetContainerTags(m.Container.Config.Env)
```

Comment:
Have you considered the performance impact of running the tag extraction this often? It would be better to do it outside the loop (as in the previous code), but still be able to work per container. Perhaps there is a way to run the filter only if a new container appears? Using a map where the key is the container (id?) and the value is the tag slice could be a way, with minimal work inside the loop.